shepherd.js/src/step.ts

```diff
@@ -604,10 +632,17 @@ export class Step extends Evented {
     // @ts-expect-error TODO: get types for Svelte components
     const content = this.shepherdElementComponent.getElement();
     const target = this.target || document.body;
+    const extraHighlightElements = this._resolvedExtraHighlightElements;
+
     target.classList.add(`${this.classPrefix}shepherd-enabled`);
     target.classList.add(`${this.classPrefix}shepherd-target`);
     content.classList.add('shepherd-enabled');
 
+    extraHighlightElements?.forEach((el) => {
+      el.classList.add(`${this.classPrefix}shepherd-enabled`);
```

Comment:
Forgive my ignorance, but why do we want to add these classes to the extra highlighted elements?

Reply:
I thought we would want the same classes applied to the target element (attackTo.element) on the extra highlighted elements as well.

Reply:
I'm not sure if we do? We just want the cutout overlay I would assume? I am not sure what the classes do anymore though.

Reply:
One of the classes I added disables the click event if `canClickTarget`option is true